source/developers-guide/csrf-protection/index.md

```diff
@@ -129,3 +129,7 @@ if (!interface_exists('\Shopware\Components\CSRFWhitelistAware')) {
 ```
 require_once __DIR__ . '/Components/CSRFWhitelistAware.php';
 ```
+
+##Possible problems for security fanatics
```

Comment:
Can you please rename this headline to a bit more polite "Troubleshooting for Http-Only Cookies" or something in that direction.